### PR TITLE
Only show user invite on org space

### PR DESCRIPTION
**Pull request by rememberlenny**

### CODE REVIEW

File: static_src/test/functional/user_invite.spec.js [hunk 1]
```diff
@@ -53,4 +58,22 @@ describe('User roles', function () {
...
+      browser.url(urlOrgXSpaceXX);
+      browser.waitForExist('.test-users');
+    });
+
+    it('should not have the user invite panel', function () {
+      userRoleElement.setAndGetUserRole(cookieManagerOrgXSpaceXX);
+      browser.url(urlOrgXSpaceXX);
+
+      browser.waitForExist('.test-users');
+      var ells = $$('.test-users-invite');
```
Comment (jcscottiii) [inline static_src/test/functional/user_invite.spec.js]: we can use the `waitForExist` with the reverse flag. http://webdriver.io/api/utility/waitForExist.html

Comment (jcscottiii) [inline static_src/test/functional/user_invite.spec.js]: that would probably help with the lint errors as well

Comment (rememberlenny) [inline static_src/test/functional/user_invite.spec.js]: Updated!

**Comment by rememberlenny** [comment]
This should go green and is finished @jcscottiii @el-mapache @fureigh

**Pull request merged**